src/utils.js

```diff
@@ -1,8 +1,11 @@
-import { resolve } from 'path';
-import { existsSync } from 'fs';
+import { resolve, dirname } from 'path';
```

Comment:
Should we stick to the same pattern we use with fs and import/use the default object rather than destructuring it?